specification/loadtestservice/resource-manager/Microsoft.LoadTestService/preview/2021-12-01-preview/loadtestservice.json

```diff
@@ -0,0 +1,469 @@
+          "LoadTests_CreateOrUpdate": {
+            "$ref": "./examples/LoadTests_CreateOrUpdate.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "LoadTests"
+        ],
+        "description": "Update a loadtest resource.",
+        "operationId": "LoadTests_Update",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LoadTestNameParameter"
+          },
+          {
+            "in": "body",
+            "name": "LoadTestResourcePatchRequestBody",
+            "description": "LoadTest resource update data",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LoadTestResourcePatchRequestBody"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- resource updated",
+            "schema": {
+              "$ref": "#/definitions/LoadTestResource"
+            }
+          },
+          "default": {
+            "description": "Resource provider error response about the failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "LoadTests_Update": {
+            "$ref": "./examples/LoadTests_Update.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "LoadTests"
+        ],
+        "description": "Delete a LoadTest resource.",
+        "operationId": "LoadTests_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LoadTestNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- resource deleted"
+          },
+          "202": {
+            "description": "Accepted -- LoadTest resource delete request accepted"
+          },
```

Comment:
Do you support Async deletes ?  Please include the "x-ms-long-running-operation" if you do. 

You can see other teams implemented this here : 
[https://github.com/Azure/azure-rest-api-specs/search?q=%22x-ms-long-running-operation%22](https://github.com/Azure/azure-rest-api-specs/search?q=%22x-ms-long-running-operation%22)

Reply:
done, thanks!